scripts/linters/pylint_extensions_test.py

```diff
@@ -3257,3 +3257,93 @@ def some_random_function(param1, param2):
 
         with self.checker_test_object.assertNoMessages():
             self.checker_test_object.checker.visit_functiondef(def_node)
+
+
+class ConcatenationCheckerTests(unittest.TestCase):
```

Comment:
Change this as per Checker class name -> `StringConcatenationCheckerTests`

Reply:
Ah, yes my bad! Thanks

Reply:
Done!!!